platform/javascript/audio_driver_javascript.cpp

```diff
@@ -152,14 +175,64 @@ void AudioDriverJavaScript::finish() {
 	/* clang-format off */
 	EM_ASM({
 		_audioDriver_audioContext = null;
+		_audioDriver_audioInput = null;
 		_audioDriver_scriptNode = null;
 	});
 	/* clang-format on */
-	memdelete_arr(internal_buffer);
-	internal_buffer = NULL;
+
+	if (internal_buffer) {
+		memdelete_arr(internal_buffer);
+		internal_buffer = NULL;
+	}
+}
+
+Error AudioDriverJavaScript::capture_start() {
+
+	input_buffer.resize(buffer_length * 2 * 8);
```

Comment:
Why multiply with 8?

Reply:
I've changed this now but the reason was that input_buffer works as a ring buffer so it needs to be larger.